gamestonk_terminal/fundamental_analysis/yahoo_finance_api.py

```diff
@@ -225,7 +225,7 @@ def sustainability(l_args, s_ticker):
         return
 
 
-# ---------------------------------------------------- CALENDAR_EARNINGS ----------------------------------------------------
+# ---------------------------------------------------- CALENDAR_EARNINGS ------------------------------------------------
```

Comment:
I think I forgot to refactor this specific file in the FA menu, because I've been removing these comments. Will do it now :)